client/csletmelearn/src/main.jsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
+import { Provider } from 'react-redux'
 import App from './App'
+import store from './store/reducer.js'
 import './index.css'
+import { GoogleOAuthProvider } from "@react-oauth/google"
 
 ReactDOM.createRoot(document.getElementById('root')).render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <Provider store={store}>
+    <GoogleOAuthProvider clientId="520624853757-rpmqbau5gpqeg7lbdfh6t5ade49og2d8.apps.googleusercontent.com">
```

Comment:
Hardcoded client-secrets - try to use .env

Reply:
Already using it , somehow missed this one. Ty